app/grandchallenge/reader_studies/models.py

```diff
@@ -1129,7 +1147,19 @@ def validate(*, creator, question, answer, images, is_ground_truth=False):
                 raise ValidationError(
                     "This user is not a reader for this study."
                 )
-
+        if (
+            question.answer_type == Question.ANSWER_TYPE_CHOICE
+            and answer not in question.options.values_list("id", flat=True)
+        ):
+            raise ValidationError(
+                "Provided option is not valid for this question"
```

Comment:
When the frontend provides an integer, is this the foreign key to a CategoricalOption object? eg, they're globally unique?

Reply:
Yes, it should be.